doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,950 @@
+
+```python
+import streamlit as st
+
+bins = st.slider(value=20, min_value=10, max_value=30, step=1, label="Bins")
+
+st.write(bins)
+```
+
+The app looks like
+
+![Streamlit Widgets Example](https://user-images.githubusercontent.com/42288570/243349378-a27fa7bd-b8dc-4b30-85f0-684a74ec40cf.png)
+
+### Panel Integer Slider Example
+
+You will find Panels input *widgets* in `pn.widgets` module.
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+bins = pn.widgets.IntSlider(value=20, start=10, end=30, step=1, name="Bins").servable()
+
+pn.pane.Str(bins).servable()
+```
+
+If you debug and inspect the code you will notice a big difference. Streamlits `bins` value is an
+`integer` while Panels `bins` value is an `IntSlider`. This is the first real indication that
+Streamlit and Panel works in very different ways.
+
+For more info about the `IntSlider` check out the
+[`IntSlider` Guide](https://panel.holoviz.org/reference/widgets/IntSlider.html).
+
+The app looks like
+
+![Panel Widgets Example](https://user-images.githubusercontent.com/42288570/243349394-084dfd83-a9fd-404e-9408-78831c2ca2e5.png)
+
+### Input Migration Steps
+
+You should
+
+- replace your Streamlit `st.some_widget` function with the corresponding Panel
+`pn.widgets.SomeWidget` class.
+
+You can identify the corresponding widget via the
+[Widgets Section](https://panel.holoviz.org/reference/index.html#widgets) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html).
+
+## Layouts
+
+*Layouts* helps you organize your *panes* and *widgets*.
+
+Panel provides layouts similar to the ones you know from Streamlit and some unique ones.
+
+Check out the [Layouts Section](https://panel.holoviz.org/reference/index.html#layouts) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html) for the full list of *layouts*.
+
+### Streamlit Layout Example
+
+```python
+import streamlit as st
+
+col1, col2 = st.columns(2)
+
+with col1:
+    st.image("https://streamlit.io/images/brand/streamlit-logo-primary-colormark-darktext.png")
+    st.write("# A faster way to build and share data apps")
+
+with col2:
+    st.image("https://panel.holoviz.org/_images/logo_horizontal_light_theme.png")
+    st.write("# The powerful data exploration & web app framework for Python")
+```
+
+The app looks like
+
+![Streamlit Layout Example](https://user-images.githubusercontent.com/42288570/243353648-add1e7af-26ba-428b-ba47-2f602ff3ce93.png)
+
+I would love to align the images (and texts) but I could not find functionality like *row*, *margin* or *spacing* to do this.
+
+### Panel Layout Example
+
+You will find Panels *layouts* in the `pn` module.
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+col1 = pn.Column(
+    pn.pane.Image("https://streamlit.io/images/brand/streamlit-logo-primary-colormark-darktext.png"),
+    pn.panel("# A faster way to build and share data apps")
```

Comment:
I don't know if it's best practice, but technically, it's not necessary to call pn.panel here.
```suggestion
   "# A faster way to build and share data apps"
```